### PR TITLE
updated the wino_bias dataset

### PR DESCRIPTION
**Pull request by JieyuZhao**
Updated the wino_bias.py script.
- updated the data_url
- added different configurations for different data splits
- added the coreference_cluster to the data features

### CODE REVIEW

File: datasets/wino_bias/wino_bias.py [hunk 1]
```diff
@@ -68,12 +71,56 @@ class WinoBias(datasets.GeneratorBasedBuilder):
...
     BUILDER_CONFIGS = [
-        datasets.BuilderConfig(
-            name="wino_bias",
-            version=VERSION,
-            description="WinoBias: Winograd-schema dataset for detecting gender bias",
+        WinoBiasConfig(
+            name='dev_type1_pro',
+            description = "winoBias dev_type1_pro_stereotype data in cornll format",
+            data_url = _URL + "/dev_type1_pro_stereotype.v4_auto_conll"
+        ),
```
Comment (lhoestq) [inline datasets/wino_bias/wino_bias.py:87]: I think we should only have the configurations "type1_pro", "type1_anti", "type2_pro" and "type2_anti".
Then each configuration can have train/dev/test splits. What do you think ?

Comment (JieyuZhao) [reply datasets/wino_bias/wino_bias.py:87]: Good point! Will update that. Thanks!

**Comment by lhoestq** [comment]
Hi @JieyuZhao ! Have you had a chance to add the different configurations ?
Thanks again for your help on this !

**Comment by JieyuZhao** [comment]
> Hi @JieyuZhao ! Have you had a chance to add the different configurations ?
> Thanks again for your help on this !

Hi @lhoestq  Yes, I've updated the code. Now the configuration will have dev/test splits.

**Comment by JieyuZhao** [comment]
> Hi @JieyuZhao ! Have you had a chance to add the different configurations ?
> Thanks again for your help on this !

Hi @lhoestq  Yes, I've updated the code. Now the configuration will have dev/test splits.

**Review by lhoestq — COMMENTED** [review]
Cool thanks !
This looks perfect this way.

Now we just need to update the dataset_infos.json (it contains the metadata of the dataset) and add dummy data to be able to test this script automatically.

To update the dataset_infos.json you just need delete the current one at `./datasets/wino_biais/dataset_infos.json`, and then run this command:
```
datasets-cli test ./datasets/wino_biais --save_infos --all_configs --ignore_verifications
```

To add the dummy data there's also a tool to add them automatically.
First delete the folder at `./datasets/wino_biais/dummy` and then run
```
datasets-cli dummy_data ./datasets/wino_biais --auto_generate --match_text_files "*conll" --n_lines 15
```

Let me know if you have questions :)
Also don't forget to run `make style` to format the code properly.

**Comment by JieyuZhao** [comment]
> Cool thanks !
> This looks perfect this way.
> 
> Now we just need to update the dataset_infos.json (it contains the metadata of the dataset) and add dummy data to be able to test this script automatically.
> 
> To update the dataset_infos.json you just need delete the current one at `./datasets/wino_biais/dataset_infos.json`, and then run this command:
> 
> ```
> datasets-cli test ./datasets/wino_biais --save_infos --all_configs --ignore_verifications
> ```
> 
> To add the dummy data there's also a tool to add them automatically.
> First delete the folder at `./datasets/wino_biais/dummy` and then run
> 
> ```
> datasets-cli dummy_data ./datasets/wino_biais --auto_generate --match_text_files "*conll" --n_lines 15
> ```
> 
> Let me know if you have questions :)
> Also don't forget to run `make style` to format the code properly.

Thanks for the instruction! I've updated the metadata and the dummy data and also do the formatting. Please let me know if more is needed. :)

**Review by lhoestq — APPROVED** [review]
Thank you !

**Pull request merged**